zerver/worker/queue_processors.py

```diff
@@ -307,6 +307,16 @@ def consume(self, data):
             # TODO: Do something smarter here ..
             pass
 
+@assign_queue('missedmessage_email_senders')
+class MissedMessageSendingWorker(EmailSendingWorker):
+    """
+    Note: Class decorators are not inherited.
+
+    This class is kept so that the events already in the queue are duly
+    processed.
```

Comment:
Oh, a third remark: because this code (or just the `@assign_queue('missedmessage_email_senders')` line) is here just for this historical reason, the comment should mention not only that it's for this historical reason but *when* the history is from. That way, in a few months someone who happens to see it can know for sure that it's OK to just delete.

Specifically, let's say something like
```
# The `missedmessage_email_senders` queue was used up through 1.7.1,
# so we keep consuming from it in case we've just upgraded from an old version.
# After the 1.8 release, we can delete it and tell admins to upgrade to 1.8 first.
```

Reply:
Cool. Assigning multiple queues to the same handler seems like right solution here. Fixing this.

Reply:
Hmm, actually, using multiple `assign_queue`s might break the logic because of this line:
```
clazz.queue_name = queue_name
```
`clazz.queue_name` will be equal to what the most recent invocation of `assign_queue` assigns to it.

Reply:
Oh, hmm, indeed -- I missed that, good catch. So we'll need two handler classes.

(The other two remarks stand, though, I think.)